src/elexmodel/models/BaseElectionModel.py

```diff
@@ -16,6 +18,28 @@
 LOG = logging.getLogger(__name__)
 
 
+def get_nat_sum_states_called(election_id, office_id):
+    year = election_id.split("-")[0]
+    nat_sum_states_called = (
+        pd.read_csv("data_for_agg_model/national_summary_states_called_{}_{}.csv".format(year, office_id))
```

Comment:
Trying to run this and print out some things so I understand what's going on with the covariance matrix - where do I get this file?